json/src/main/java/org/openx/data/jsonserde/json/JSONObject.java

```diff
@@ -1426,7 +1449,7 @@ String toString(int indentFactor, int indent) throws JSONException {
      * @throws JSONException If the value is or contains an invalid number.
      */
     public static String valueToString(Object value) throws JSONException {
-        if (value == null ) {
+        if (value == null || value == NULL) {
```

Comment:
Oh!  This is because NULL is actually a JSONObject instantiated to represent a JSON null behaviorally.  So, they're quantitatively different, but they both mean the same thing for our purposes here.  It's not you! :-D